### PR TITLE
doing homework5

### PR DESCRIPTION
**Pull request by Hromy-Ann**

### CODE REVIEW

File: src/main/java/org/example/homework5/Task1.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        int evenDigitSum = getEvenDigitSum(input);
+        System.out.println("For input number " + input + " sum of even digits is " + evenDigitSum);
+    }
+
+    private static int getEvenDigitSum(int number) {
+        if (number < 0) {
+            return -1;
+        }
+        int result = 0;
+        int lessDigit = number;
```
Comment (alexshamrai) [inline src/main/java/org/example/homework5/Task1.java]: I don't really understand what is `lessDigit`?)
Looks like you can just keep on using `number`, isn't it?

File: src/main/java/org/example/homework5/Task2.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    private static void printArray(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            System.out.println("Element " + i + " contents " + array[i]);
+        }
+    }
+
+    private static int[] sortIntegers(int[] array) {
+        int[] resultSort = Arrays.copyOf(array, array.length);
+        //Selection sort implementation
```
Comment (alexshamrai) [inline src/main/java/org/example/homework5/Task2.java:32]: What kind of section do you expect in `sortIntegersMethod`? )))

**Review by alexshamrai — APPROVED** [review]

File: src/main/java/org/example/homework5/Task1.java (hunk 1 above)
Comment (Hromy-Ann) [reply src/main/java/org/example/homework5/Task1.java]: This variable was there to keep input parameter unchanged. As it was uselesss in thismethod I've refactoredcode to remove it

File: src/main/java/org/example/homework5/Task2.java (hunk 2 above)
Comment (Hromy-Ann) [reply src/main/java/org/example/homework5/Task2.java:32]: "Selection sort" is the name of sorting algorithm that I selected to implement

Comment (alexshamrai) [reply src/main/java/org/example/homework5/Task2.java:32]: I got it, you can leave it for yourself, but for me that's kind of info does not bring a value. Up to you, in our reality that's not a big deal

**Pull request merged**